quotes/data.json

```diff
@@ -332,13 +334,21 @@
             "color": "#9B59B6",
             "name": "Triage Team"
         },
+        "collaborator": {
+            "color": "#1F8B4C",
+            "name": "Collaborator"
+        },
         "docs_triage": {
             "name": "Docs Triage"
         },
         "superhelper": {
-            "color": "#c27c0e",
+            "color": "#ee5beb",
             "name": "superhelper"
         },
+        "retrogradle": {
+            "color": "#15e4c7",
+            "name": "Retrogradle"
+        },
```

Comment:
`retrogradle` is below `contributor`, according to the roles list (as viewed through Discord Mobile).